src/index.html

```diff
@@ -2,13 +2,46 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav__wrapper">
+        <div class="nav__logo">
+          <img src="images/logo.png" alt="moyo logo">
+        </div>
+        <nav class="nav__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link" >Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">SMARTPHONES</a>
+            </li>
+            <li class="nav__item">
+              <a data-qa="hover" class="nav__link">Laptops & computers</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
```

Comment:
will be more readability if you add spaces between childs
```suggestion
            <li class="nav__item">
              <a href="#" class="nav__link is-active">Apple</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link" >Samsung</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link">SMARTPHONES</a>
            </li>

            <li class="nav__item">
              <a data-qa="hover" class="nav__link">Laptops & computers</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link">Gadgets</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link">Tablets</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link">Photo</a>
            </li>

            <li class="nav__item">
              <a href="#" class="nav__link">Video</a>
            </li>
```